gui-js/apps/minsky-electron/src/app/managers/WindowManager.ts

```
@@ -131,6 +132,66 @@ export class WindowManager {
+  
+  /// wrappers around the standard electron dialogs that saves the directory opened as a defaultPath
+  /// if options.defaultPath is set to either models or data.
+  static async showOpenDialog(...args: any[])
+  {
+    let options=args[args.length-1] as OpenDialogOptions;
+    let defaultDirectoryKey=this.processDefaultDirectory(options);
+
+    if (args.length>1)
+      var res=await dialog.showOpenDialog(args[0],options);
+    else
+      var res=await dialog.showOpenDialog(options);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'res'. Consider to delete it or rename it.

'res' is defined here:


(lint/suspicious/noRedeclare)